tensorflow/contrib/rnn/kernels/lstm_ops_gpu.cu.cc

```diff
@@ -0,0 +1,20 @@
+#if GOOGLE_CUDA
+
+#define EIGEN_USE_GPU
+
+#include "tensorflow/contrib/rnn/kernels/lstm_ops.h"
+
+namespace tensorflow {
+
+namespace functor {
+
+typedef Eigen::GpuDevice GPUDevice;
+
+template struct TensorMemZero<GPUDevice, float>;
```

Comment:
Can you add support for double too, in your kernels? i.e., via templates? In general, let's add support on float and double at least; possibly in the future also quantized ints for purposes of working on mobile devices.